DQMOffline/Trigger/plugins/RazorMonitor.h

```diff
@@ -0,0 +1,120 @@
+// -----------------------------
+//
+// Offline DQM for razor triggers. The razor inclusive analysis measures trigger efficiency
+// in SingleElectron events (orthogonal to analysis), as a 2D function of the razor variables
+// M_R and R^2. Also monitor dPhi_R, used offline for  QCD and/or detector-related MET tail
+// rejection.
+// Based on DQMOffline/Trigger/plugins/METMonitor.*
+//
+// ----------------------------- 
+#ifndef RAZORMONITOR_H
```

Comment:
please add the package name to the include guard eg,

DQMOFFLINE_TRIGGER_RAZORMONITOR_H